exchange/src/exchange/traders/trader_types.hpp

```diff
@@ -1,20 +1,31 @@
 #pragma once
 
+#include <stdexcept>
 #include <string>
 #include <unordered_map>
 #include <variant>
 
 namespace nutc {
 namespace manager {
 
+enum TraderType { REMOTE, LOCAL, BOT };
+
 struct generic_trader_t {
```

Comment:
STEEV

STRUCTS ARE FOR POD

STRUCTS DON'T HAVE METHODS

Reply:
sorry my rust days make me forget

Reply:
also POD should end with `_t`

classes should be `PascalCase` with no `_t`